new-lamassu-admin/src/components/LogsDownloaderPopper.js

```diff
@@ -256,10 +281,18 @@ const LogsDownloaderPopover = ({ name, query, args, title, getLogs }) => {
                 />
               </div>
             )}
+            <div className={classes.radioButtonsContainer}>
+              <RadioGroup
+                name="simplified-tx-logs"
+                value={selectedAdvancedRadio}
+                options={radioButtonOptions.slice(2, 4)}
+                ariaLabel="simplified-tx-logs"
+                onChange={handleAdvancedRadioButtons}
+                className={classes.radioButtons}
+              />
+            </div>
```

Comment:
Can't this be handled by the external component? Not every log download will have simplified and advanced options